src/client/elm/Transaction/Event.elm

```diff
@@ -294,6 +296,31 @@ type alias ProtocolUpdate =
     }
 
 
+type alias Description =
+    { name: String
+    , url: String
+    , description: String
+    }
+
+
+-- Identification number of an anonymity revoker or identity provider
```

Comment:
I think these should be doc comments. [Format described here](https://package.elm-lang.org/help/documentation-format)